apps/examples/mediaplayer/mediaplayer_main.cpp

```diff
@@ -45,6 +62,9 @@ static const int TEST_AAC = 2;
 static const int TEST_OPUS = 3;
 static const int TEST_WAVE = 4;
 static const int TEST_BUFFER = 5;
+static const int TEST_HTTP = 6;
+
+static const std::string TEST_HTTP_URL = "http://ra01.sycdn.kuwo.cn/resource/n3/32/56/3260586875.mp3";
```

Comment:
please change this to blank one as we discussed before.

Reply:
Fixed, Thanks.